modules/store/src/store_module.ts

```diff
@@ -149,12 +149,16 @@ export class StoreModule {
   ): ModuleWithProviders;
```

Comment:
Do these changes in a separate commit

Reply:
Hey Brandon, these changes are happening automatically as per the precommit script. Any suggestions on how to stop it.

Reply:
Don't worry about it then. Must have merged another change that didn't get formatted.